mirascope/core/bedrock/call_response_chunk.py

```diff
@@ -87,3 +93,21 @@ def output_tokens(self) -> int | None:
         if self.usage:
             return self.usage["outputTokens"]
         return None
+
+    @property
+    def common_finish_reasons(self) -> list[types.FinishReason] | None:
+        return _convert_finish_reasons_to_common_finish_reasons(
+            cast(list[str], self.finish_reasons)
+        )
+
+    @property
+    def common_usage(self) -> Usage | None:
+        if self.input_tokens is None and self.output_tokens is None:
```

Comment:
why have a shared utility for the finish reason conversion but not he usage creation?

is there any reason not to just have shared utilities for all of these? Even if it's just a simple cast line so that any changes in the future happen only in one place?